Bugzilla/Auth.pm

```diff
@@ -111,6 +111,26 @@ sub login {
         });
     }
 
+    my $mfa_group = 'admin';
+    my %mfa_allowed_page = map { $_ => 1 } qw(index.cgi post_bug.cgi process_bug.cgi userprefs.cgi);
+    if (   !i_am_webservice()
+        && $user->id
+        && !$user->mfa
+        && $mfa_group
+        && $user->in_group($mfa_group) )
+    {
+        my $cgi = Bugzilla->cgi;
+        my $template = Bugzilla->template;
+        my $script = $cgi->script_name;
+        $script =~ s{^/}{};
+
+        unless ($mfa_allowed_page{$script}) {
```

Comment:
Per above, encourage replacing with 'userprefs.cgi?tab=mfa only', all other requests refused. (Even index.cgi, because that'll just direct them to ?tab=mfa anyways, as intended.)